apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class ConnectionPoolOptions
```

Comment:
I think its ok for these properties to change in realtime.  Thread-wise, it will be safe to set individual options without atomicity. The effects will not be immediate if say, maxsessions is reduced with an outstanding count greater than the new value.   But I think thats ok.   I don't see a reason to put a builder pattern here, but happy to discuss further.  Builder pattern always struck me as more java-esque with lower usability.